docs(tutorials): fixed various tutorials MTA-1462 (#90)
scaleway/docs-content: tutorials/odoo/index.mdx

---
layout: docs
title: Deploying Odoo ERP on Ubuntu 20.04 LTS (Focal Fossa)
description: Learn how to deploy Odoo, a versatile business management software on Ubuntu 20.04 LTS (Focal Fossa)
tags: odoo, erp, cms, business management
excerpt: |
  Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the companies GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services. 
  The modular design of Odoo allows developers to create additional modules or apps and distribute them on the Odoo marketplace, which provides more than 20.000 modules. This fact makes Odoo a versatile and customizable solution that can be adapted to many usage scenarios. 

>   This tutorial will show you how to deploy Odoo 14 on a Scaleway Instance or Scaleway Dedibox dedicated server, running on Ubuntu Focal Fossa (20.04 LTS).

Reviewer: ```suggestion
  This tutorial shows you how to deploy Odoo 14 on a Scaleway Instance or Scaleway Dedibox dedicated server, running on Ubuntu Focal Fossa (20.04 LTS).
```

The modular design of Odoo allows developers to create additional modules or apps and distribute them through the Odoo marketplace, which provides more than 20,000 modules. This makes Odoo a versatile and customizable solution that can be adapted to many usage scenarios.

This tutorial will show you how to deploy Odoo 14 on a Scaleway machine running Ubuntu Focal Fossa (20.04 LTS). This could be a [Dedibox](https://www.scaleway.com/en/dedibox/), an [Instance](https://www.scaleway.com/en/virtual-instances/) or a [Bare Metal server](https://www.scaleway.com/en/bare-metal-servers/).

<Message type="requirement">

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
- You have one of the following remote machines running Ubuntu Focal Fossa (20.04 LTS):

>   - An [Instance](/compute/instances/how-to/create-an-instance)

Reviewer: ```suggestion
  - An [Instance](/compute/instances/how-to/create-an-instance/)
```

---
layout: docs
title: Deploying Odoo ERP on Ubuntu 20.04 LTS (Focal Fossa)
description: Learn how to deploy Odoo, a versatile business management software on Ubuntu 20.04 LTS (Focal Fossa)
tags: odoo, erp, cms, business management
excerpt: |

>   Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the companies GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services. 

Reviewer 2: ```suggestion
  Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the company's GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services. 
```

    - **Country:** Select your country from the drop-down list
    - **Demo data:** Tick this box to fill up the Odoo database with demo data. This feature can be useful for testing the application.

  Once all data is filled in, click **Create database** to create the Odoo database and your first login. The application dashboard displays:

  <Lightbox src="scaleway-odoo_app_dashboard.png" width="900" alt="" />

> 3. (Optional): Click the **Install** button next to each app you want to install it automatically on your Odoo server. You can see more details about each application by clicking on the **Learn More** button.

Reviewer 2: ```suggestion
3. (Optional): Click the **Install** button next to each app you want to install automatically on your Odoo server. You can see more details about each application by clicking on the **Learn More** button.
```

  }
  ```

4. Activate the site by creating a symlink in the `sites-enabled` directory:

  ```
  ln -s /etc/nginx/sites-available/odoo.example.com /etc/nginx/sites-enabled/
  ```

> 5. Run `certbot` to request a new certificate and configure Nginx. Follow the [instructions](/tutorials/configure-nginx-letsencrypt/#installing-certbot-and-obtaining-a-certificate) given during the certificate request:

Reviewer: ```suggestion
5. Run `certbot` to request a new certificate and configure Nginx. Follow the [instructions](/tutorials/configure-nginx-letsencrypt/#installing-certbot-and-obtaining-a-certificate/) given during the certificate request:
```
